src/editor/nodes/KitPieceNode.js

```diff
@@ -34,18 +34,13 @@ export default class KitPieceNode extends EditorNodeMixin(Model) {
         const loopAnimationComponent = json.components.find(c => c.name === "loop-animation");
 
         if (loopAnimationComponent && loopAnimationComponent.props) {
-          const { clip, activeClipIndex } = loopAnimationComponent.props;
-          const { activeClipIndices } = loopAnimationComponent.props;
+          const { clip, activeClipIndices } = loopAnimationComponent.props;
 
-          if (activeClipIndices !== undefined) {
-            node.activeClipIndices = activeClipIndices;
+          if (clip !== undefined && node.model && node.model.animations) {
+            // DEPRECATED: Old loop-animation component stored the clip name rather than the clip index
+            node.activeClipIndices = [node.model.animations.findIndex(animation => animation.name === clip)];
```

Comment:
Is `-1` a valid clip index in case the model doesn't have a matching `animation.name`?

Reply:
Yeah, that's definitely a bug, I'll fix it. Thanks!